### ISSUE TITLE
Contents have no heading at high zoom levels

### ISSUE THREAD

**Issue by andrewhick** [alphagov/wcag-primer#136]
When the text size is increased to 200%, page zoomed to 400% or mobile view, the table of contents modal does not have a heading.
